index.html

```diff
@@ -259,6 +270,134 @@ <h3>Video Content Hints</h3>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>maintain-framerate</code></dfn></td>
+              <td>
+                <p>Degrade resolution in order to maintain framerate.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn data-idl><code>maintain-resolution</code></dfn></td>
+              <td>
+                <p>Degrade framerate in order to maintain resolution.</p>
+              </td>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>balanced</code></dfn></td>
+              <td>
+                <p>Degrade a balance of framerate and resolution.</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      An attribute is defined for RTCRtpSendParameters that allows this
+      to be explicitly indicated for an RTCRtpSender:
+
+      <pre class='idl'>partial dictionary RTCRtpSendParameters {
+        RTCDegradationPreference degradationPreference;
+       };</pre>
+      <section>
+        <h2>Dictionary <code><a>RTCRtpSendParameters</a></code> New Members</h2>
+        <dl>
+          <dt data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>degradationPreference</code></dfn> of type
+            <span class="idlMemberType"><a>RTCDegradationPreference</a></span>,
+            defaulting to <code>"balanced"</code></dt>
+          <dd>
+            <p>When bandwidth is constrained and the
+              <code><a>RTCRtpSender</a></code> needs to choose between degrading
+              resolution or degrading framerate,
+              <code>degradationPreference</code> indicates which is
+              preferred.</p>
+          </dd>
+        </dl>
+      </section>
+    <section>
+      <h3>Behavior of an RTCPeerConnection</h3>
+      <p>An <a>RTCRtpSender</a> transmitting a MediaStreamTrack for which a
+        contentHint attribute has been
+        set MUST use the following degradation preferences, unless an
+        explicit degradationPreference attribute has been set in the sender's
+        parameters:
+      </p>
+      <ul>
+        <li>For a video track with the attribute value "motion",
+          use "maintain-framerate".
+        </li>
+        <li>For a video track with the attribute value "detail",
+          use "maintain-resolution".
+        </li>
+        <li>For a video track with the attribute value "text",
+          use "maintain-resolution". In addition, if the
+        encoding codec is AV1, activate encoding tools for "text" mode.
+        </li>
+      </ul>
+    </section>
+    <section>
+      <h3>Behavior of a MediaStreamRecorder</h3>
+      <p>For a video track with the attribute value "text", if the
+        encoding codec is AV1, activate encoding tools for "text"
+        mode.
```

Comment:
Same question here: is this determination to be made once per recording in [`start()`](https://w3c.github.io/mediacapture-record/MediaRecorder.html#dom-mediarecorder-start), or updated continuously or on some frequency? Does AV1 support changing this during encoding?

Also, the MediaRecorder spec allows recording of multiple video tracks in the same stream (even most implementations only support one video track AFAIK). If two or more video tracks with competing contentHints are being recorded, which hint(s) affects encoding?

Reply:
Multiple video tracks would run through multiple encoders, so the contentHints would affect each encoder separately.

As for what AV1 supports changing, I'd make this a new issue and assign it to @aboba - AV1 is still a bit of a black box to me.